bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+info()
+{
+    echo "Info: $*"
+}
+
+warn()
+{
+    echo "Warning: $*"
+}
+
+fatal()
+{
+    echo "Fatal: $*"
+}
+
+wait_for_api()
+{
+  local n=20
+  # In the case of a long migration, we should fix this loop or ensure docs clarify user will have to start horti and medic-sentinel
+  # after API container finishes.
```

Comment:
Yeah I'm not sure this is going to work. I think we need an infinite retry somewhere, rather than just giving up. What's the best way to do this?

Reply:
Yup, it's handled in Sentinel with an infinite loop. I'm not sure we'll need that here, as it depends on how the horti rework looks.